http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/WsmDao.scala

```diff
@@ -117,10 +125,29 @@ final case class CreateStorageContainerRequest(workspaceId: WorkspaceId,
 final case class CreateStorageContainerResult(resourceId: WsmControlledResourceId)
 final case class WorkspaceDescription(id: WorkspaceId,
                                       displayName: String,
+                                      spendProfile: String,
                                       azureContext: Option[AzureCloudContext],
                                       gcpContext: Option[GoogleProject]
 )
 
+//Landing Zone models
+final case class LandingZone(landingZoneId: UUID,
+                             billingProfileId: UUID,
+                             definition: String,
+                             version: String,
+                             createdDate: String
+)
+final case class ListLandingZonesResult(landingZones: List[LandingZone])
+
+final case class LandingZoneResource(resourceId: String,
+                                     resourceType: String,
+                                     resourceName: String,
+                                     resourceParentId: String,
+                                     region: String
+)
+final case class LandingZoneResourcesByPurpose(purpose: String, deployedResources: List[LandingZoneResource])
```

Comment:
It's an enum, or rather 2 enums from (here)[https://github.com/DataBiosphere/terra-landing-zone-service/blob/main/service/src/main/java/bio/terra/landingzone/library/landingzones/deployment/ResourcePurpose.java] and (here)[https://github.com/DataBiosphere/terra-landing-zone-service/blob/main/service/src/main/java/bio/terra/landingzone/library/landingzones/deployment/SubnetResourcePurpose.java]. I tried importing the LandingZoneService Library to use them directly, but am running into issues since it's published to (jfrog libs-snapshot-local)[https://broadinstitute.jfrog.io/ui/native/libs-snapshot-local] but not to libs-release!

Reply:
maybe just create model in Leo like this
```
sealed trait LzResourcePurpose extends Serializable with Product {
  def asString: String
}
object LzResourcePurpose {
  final case class Shared extends LzResourcePurpose {
    def asString: String = "SHARED_RESOURCE" 
  }
  final case class Wlz extends LzResourcePurpose {
    def asString: String = "WLZ_RESOURCE" 
  }
}
```

Reply:
I ended up creating a very simple enum and making a [linked ticket](https://broadworkbench.atlassian.net/browse/TOAZ-269) to migrate to using the landing zone enum once the landing zone library makes it to release.

Reply:
Did that in my most recent commit